docs/source/conf.py

```diff
@@ -205,7 +205,7 @@ def skip(app, what, name, obj, skip, options):
 
 
 def setup(app):
-    app.add_stylesheet('css/pyro.css')
+    app.add_css_file('css/pyro.css')
```

Comment:
`add_stylesheet` is renamed to `add_css_file` since version [1.8](https://www.sphinx-doc.org/en/master/extdev/appapi.html#sphinx.application.Sphinx.add_css_file)

Reply:
I'm not sure. Probably you got some sort of circular import issues (e.g. `from ... import *` statement or type annotation...)